diff --git a/rust-pty/Cargo.toml b/rust-pty/Cargo.toml
index fc67a30..3b24d18 100644
--- a/rust-pty/Cargo.toml
+++ b/rust-pty/Cargo.toml
@@ -15,7 +15,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 # https://github.com/wezterm/wezterm/issues/6783
-portable-pty = { version = "0.8.1", features = ["serde_support"] }
+portable-pty = { version = "0.9.0", features = ["serde_support"] }
 lazy_static = "1.4"
 crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
 serde = { version = "1.0", features = ["derive"] }
